index.d.ts

```diff
@@ -238,70 +238,62 @@ declare namespace fileType {
 	interface FileTypeResult {
 		/**
 		One of the supported [file types](https://github.com/sindresorhus/file-type#supported-file-types).
-		*/
+		 */
 		ext: FileType;
 
 		/**
 		The detected [MIME type](https://en.wikipedia.org/wiki/Internet_media_type).
-		*/
+		 */
 		mime: MimeType;
 	}
 
 	type ReadableStreamWithFileType = ReadableStream & {
 		readonly fileType?: FileTypeResult;
 	};
-}
 
-declare const fileType: {
 	/**
-	Detect the file type of a `Buffer`/`Uint8Array`/`ArrayBuffer`. The file type is detected by checking the [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)#Magic_numbers_in_files) of the buffer.
+	Detect the file type of a `Buffer`/`Uint8Array`/`ArrayBuffer`.
+  The file type is detected by checking the [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)#Magic_numbers_in_files) of the buffer.
```

Comment:
Don't do unrelated changes.

Reply:
The reason changed it, is because we are starting go beyond the "_magic numbers_". I will revert.